app/jobs/update_account_invoices.rb

```diff
@@ -37,9 +37,7 @@ def update(account_invoice)
       if billable_periods.any?
         oldest_enterprise = billable_periods.first.enterprise
         address = oldest_enterprise.address.dup
-        first, _space, last = (oldest_enterprise.contact || "").partition(' ')
         address.update_attributes(phone: oldest_enterprise.phone) if oldest_enterprise.phone.present?
-        address.update_attributes(firstname: first, lastname: last) if first.present? && last.present?
         account_invoice.order.update_attributes(bill_address: address, ship_address: address)
       end
 
```

Comment:
@oeoeaio the name of the enterprise contact is being applied here to account invoices? I decided to take this out...

Reply:
@Matt-Yorkley this is a good example of where we need the `contact` to address the invoice to someone.